.github/workflows/CD.yml

```diff
@@ -0,0 +1,40 @@
+name: CD
```

Comment:
Should we have a more descriptive name like `um2nc CD`?

Reply:
That's probably covered because we're in the `um2nc-standalone` repo, I think

Reply:
I'm not sure it is necessary as `um2nc` is our only "product" being deployed from the repo. In general we've not done this for other repositories.